docs/index.md

```diff
@@ -2,45 +2,42 @@
-- Go over the workflow documentation below and read the relevant sections(s).
+## Event Attendees
 
-**********************
+- Join the event Slack team recommended by event organizers, this is often the [DataRefuge Slack](https://rauchg-slackin-qonsfhhvxs.now.sh/). During the event people share expertise and answer each other's questions here.  
+- Pick your role from the paths below, get account credentials, and make sure you have access to the key documents and tools you need to work. Organizers will instruct you on these steps.
+- Review the relevant sections(s) of this workflow.
 
-## Overview
+### Path I. Website Archiving
 
-### 1. [Seeding](seeding.md)
+#### [Seeding](seeding.md)
 
 Seeders canvass the resources of a given government agency, identifying important URLs. They identify whether those URLs can be crawled by the Internet Archive's webcrawler. If the URLs are crawlable, the Seeders nominate them to the End-of-Term (EOT) project. Otherwise, they add them to the Archivers app using the project's Chrome Extension.
 
-### 2. [Researching](researching.md)
+### Path II. Archiving More Complex Datasets
+### A. [Researching](researching.md)
 
 Researchers inspect the "uncrawlable" list to confirm that Seeders' assessments were correct (that is, that the URL/dataset is indeed uncrawlable), and investigate how the dataset could be best harvested. [Researching.md](researching.md) describes this process in more detail.
 
 *We recommend that Researchers and Harvesters (see below) work together in pairs, as much communication is needed between the two roles. In some cases, one person will fulfill both roles.*
 
-### 3. [Harvesting](harvesting.md)
+### B. [Harvesting](harvesting.md)
 
 Harvesters take the "uncrawlable" data and try to figure out how to actually capture it based on the recommendations of the Researchers. This is a complex task which can require substantial technical expertise and different techniques for different tasks. Harvesters should also review the [Harvesting Toolkit](https://github.com/edgi-govdata-archiving/harvesting-tools) for tools.
 
-### 4. Checking
-
-**Note: This role is currently performed by the Baggers, and does not exist separately.**
-
-Checkers inspect a harvested dataset and make sure that it is complete. The main question the checkers need to answer is "will the bag make sense to a scientist"? Checkers need to have an in-depth understanding of harvesting goals and potential content variations for datasets.
+### C. [Checking/Bagging](bagging.md)
 
-### 5. [Bagging](bagging.md)
+Checkers inspect a harvested dataset and make sure that it is complete. The main question the checkers need to answer is "will the bag make sense to a scientist"? Checkers need to have an in-depth understanding of harvesting goals and potential content variations for datasets. <br /> **Note: Checking is currently performed by Baggers and does not exist separately.**
```

Comment:
"currently peformed by Baggers and does not exist as a separate stage in the app"?